src/views/v2/emails/template-parts/body/greeting.php

```diff
@@ -0,0 +1,32 @@
+<?php
+/**
+ * Event Tickets Emails: Main template > Body > Greeting.
```

Comment:
Maybe we can rename this into something more generic? It could be a greeting or it could be something else. Also, not sure if it's applicable to all emails. If it's related to a particular email it should live in the folder corresponds to that email.

Reply:
I feel `greeting` is an appropriate name. The only other name I can think of would be `welcome`. In any case, I moved it into the `purchase-receipt` folder since it's specifically for that email only.

Reply:
Thanks. Yes, I was thinking more about `intro-text`, `description`, or something that's not only related to a greeting/welcome as the content of the text can vary.

Reply:
I like `intro`. Changing it to that.